### ISSUE TITLE
Button focus styles are not visible enough

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyfive#439]
**Description**

The focus outline needs to be 2px solid, it is now 1px dotted.
On the filled button, the colors change on focus, which is helpful for users who can rely on color. This color change is removed in 
#414.

**Step-by-step reproduction instructions**
If your browser does not have focus styles on tabbing enabled, enable it now.

Add a group block with the section style that has the purple background.
Place a filled button and an outlined button.
**Remember to add the links.**
Save.
View the buttons on the front of the site.
Place the cursor in the browser address bar and navigate to the buttons using the tab key.
Confirm if you can see when the button is focused:

![image](https://github.com/user-attachments/assets/202fc554-0a8d-4ab7-9fa3-b0d2befa2071)

**Expected behavior**

A clearly visible focus style.

**Comment by carolinan** [comment]
Suggested solution:
Remove the focus styles from the button in theme.json except for the offset:
![image](https://github.com/user-attachments/assets/e3020f3d-e70e-4934-b1e5-d9836d5fce77)

Or possibly even increase the offset:
![image](https://github.com/user-attachments/assets/75a869b1-7484-4d5a-9684-f1d9b88e317d)
